#!/usr/bin/env python

PACKAGE = "me5413_world"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("speed_target", double_t, 1, "Default: 0.5[m/s]", 0.5, 0.1, 1.0)

gen.add("LQR_K", double_t, 1, "Default: 0.1", 0.1, 0, 10.0)

exit(gen.generate(PACKAGE, "lqr_tracker_node", "lqr_tracker"))
